File: C/1015_Distance_Between_Two_Points.c
#include <stdio.h>
#include <math.h>

int main()
{
    double a, b, x, y;
    scanf("%lf %lf %lf %lf", &a, &b, &x, &y);
    printf("%.4f\n", sqrt(pow(x - a, 2) + pow(y - b, 2)));
    return 0;
}

File: Python/1015_Distance_Between_Two_Points.py
import math

a = input().split()
b = input().split()

x1 = float(a[0])
y1 = float(a[1])
x2 = float(b[0])
y2 = float(b[1])

distance = math.sqrt((pow(x2 - x1, 2) + (pow(y2 - y1, 2))))

print(f"{distance:.4f}")